Jenkinsfile

```diff
@@ -111,6 +111,12 @@ podTemplate(name: podName,
                         ttyEnabled: true,
                         command: 'cat',
                         privileged: true,
+                        workingDir: '/workDir'),
+                containerTemplate(name: 'ostree-boot-image',
+                        image: DOCKER_REPO_URL + '/' + OPENSHIFT_NAMESPACE + '/ostree-boot-image:' + SLAVE_TAG,
```

Comment:
should have tag for your image...OSTREE_BOOT_IMAGE_TAG

Reply:
ok - fixing